specification/apimanagement/resource-manager/Microsoft.ApiManagement/preview/2020-06-01-preview/definitions.json

```diff
@@ -5901,6 +5901,112 @@
           "description": "UTC Timestamp when the service was soft-deleted. The date conforms to the following format: yyyy-MM-ddTHH:mm:ssZ as specified by the ISO 8601 standard."
         }
       }
+    },
+    "PortalRevisionCollection": {
+      "properties": {
+        "value": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/PortalRevisionContract"
+          },
+          "description": "Collection of portal revisions.",
+          "readOnly": true
+        },
+        "count": {
+          "type": "integer",
+          "format": "int64",
+          "description": "Total record count number across all pages."
+        },
+        "nextLink": {
+          "type": "string",
+          "description": "Next page link, if any.",
+          "readOnly": true
+        }
+      },
+      "description": "Paged list of portal revisions."
+    },
+    "PortalRevisionContract": {
+      "properties": {
+        "properties": {
+          "x-ms-client-flatten": true,
+          "$ref": "#/definitions/PortalRevisionContractProperties",
+          "description": "Properties of the portal revisions."
+        }
+      },
+      "allOf": [
+        {
+          "$ref": "./apimanagement.json#/definitions/Resource"
+        }
+      ],
+      "description": "Portal revisions contract details."
+    },
+    "PortalRevisionContractProperties": {
+      "properties": {
+        "id": {
+          "type": "string",
+          "description": "Portal revision identifier.",
+          "minLength": 1,
+          "maxLength": 256
+        },
+        "description": {
+          "type": "string",
+          "description": "Portal revision description.",
+          "maxLength": 2000
+        },
+        "statusDetails": {
+          "type": "string",
+          "description": "Portal revision publishing status details.",
+          "maxLength": 2000
+        },
+        "status": {
+          "type": "string",
+          "description": "Portal revision publishing status",
+          "enum": [
+            "pending",
+            "publishing",
+            "completed",
+            "failed"
+          ],
+          "x-ms-enum": {
+            "modelAsString": true,
+            "name": "portalRevisionStatus",
+            "values": [
+              {
+                "value": "pending",
+                "description": "Portal revision publishing is pending"
+              },
+              {
+                "value": "publishing",
+                "description": "Portal revision is publishing"
+              },
+              {
+                "value": "completed",
+                "description": "Portal revision publishing completed"
+              },
+              {
+                "value": "failed",
+                "description": "Portal revision publishing failed"
+              }
+            ]
+          }
+        },
+        "isCurrent": {
```

Comment:
>isCurrent [](start = 9, length = 9)

How does "current" in `isCurrent` related to the revision being "public" as the description states?

Reply:
The portal revision represents the state of the developer portal. If the revision has flag isCurrent=true it means the changes in this revision visible on the public developer portal website.

Reply:
Makes sense. Thanks for explaining.